base/linalg/lapack.jl

```diff
@@ -31,6 +31,12 @@ macro chkuplo()
       throw(ArgumentError(string("uplo argument must be 'U' (upper) or 'L' (lower), got ", repr(uplo)))))
 end
 
+#Check that {c}transpose is correctly specified
+macro chktrans()
+    :((trans=='N' || trans=='C' || trans=='T') ||
+      throw(ArgumentError(string("trans argument must be 'N' (no transpose), 'T' (transpose), or 'C' (conjugate transpose), got ", repr(trans)))))
+end
```

Comment:
I wrote `chkuplo` a long time ago as one of my first macros. It shouldn't be considered sacred by any means.

I tried passing in the variable name as a parameter in `chkuplo` in a previous try, but it didn't work for me. I think there were things I didn't escape and quote correctly.